packages/ListBox/src/hooks/hooks.js

```diff
@@ -111,43 +111,40 @@ export function useOnScrolled() {
 
     if ($box && $option) {
       const rectBox = $box.getBoundingClientRect();
+      const scrollTop = $box.scrollTop;
+      const { marginBottom, marginTop } = window.getComputedStyle($option);
+      const marginBottomNumber = Number.parseInt(marginBottom, 10);
+      const marginTopNumber = Number.parseInt(marginTop, 10);
       const rectOption = $option.getBoundingClientRect();
 
-      // if options its between the box don't do anything
-      if (rectOption.top >= rectBox.top && rectOption.bottom <= rectBox.bottom) {
+      const nextTopPositionForOption =
+        rectOption.height * state.activeOption + state.activeOption * (marginBottomNumber + marginTopNumber);
+
+      // this occurs when the first element is a Divider so we want to scroll to the top
+      // so the user can see the Divider text instead to see only the option
+      if (state.activeOption - 1 === 0 && state.options[state.activeOption - 1].isDisabled) {
```

Comment:
🤔 it might be that all of them are disabled but at that point, you couldn't use the up/down arrow, is common to have the first element to be a Divider (group) and unlikely you have two separators at the beginning. Could happen just doesn't make sense from UI/UX perspective.